src/main/java/seedu/address/model/client/Birthday.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.client;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+/**
+ * Represents a client's birthday in MyInsuRec.
+ */
+public class Birthday {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Birthday should be in the format DDMMYYYY";
```

Comment:
Just a UX issue, maybe a better can be just `DDMM`? Reason being remembering someone's birthday doesn't require the year he/she was born

Reply:
Hmm, for this i think it is ok to leave it as `DDMMYYYY`, coz usually like when give bday card, they sometimes might write the age down also.

Reply:
I think actually it is a security issue to store the entire birthday (sensitive data), so if everyone agrees I will go ahead and change to DDMM.